default/content/settings.json

```diff
@@ -111,16 +112,17 @@
         "font_scale": 1,
         "blur_strength": 10,
         "shadow_width": 2,
+        "letterbox_size": 8,
         "main_text_color": "rgba(220, 220, 210, 1)",
         "italics_text_color": "rgba(145, 145, 145, 1)",
         "underline_text_color": "rgba(188, 231, 207, 1)",
         "quote_text_color": "rgba(225, 138, 36, 1)",
         "chat_tint_color": "rgba(23, 23, 23, 1)",
+        "chat_letterbox_color": "rgba(0, 0, 0, 1)",
         "blur_tint_color": "rgba(23, 23, 23, 1)",
         "user_mes_blur_tint_color": "rgba(30, 30, 30, 0.9)",
         "bot_mes_blur_tint_color": "rgba(30, 30, 30, 0.9)",
         "shadow_color": "rgba(0, 0, 0, 1)",
-        "waifuMode": false,
```

Comment:
### **Who allowed you to remove the word "waifu" from my silly tavern?** 😠😡

Reply:
Blame genericness :p